benchmarks/common/include/benchmark/common/benchmark_getter.hpp

```diff
@@ -0,0 +1,53 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s)
+#include "benchmark_vector.hpp"
+
+namespace algebra::bench_op {
+
+// Functions to be benchmarked
+
+struct phi {
+  inline static const std::string name{"phi"};
+  template <typename vector_t>
+  auto operator()(const vector_t &a) const {
+    return algebra::getter::phi(a);
+  }
+};
```

Comment:
For these, maybe defining a macro and then instantiating a bunch of those macro instances would be a cleaner way of doing this?